src/search.rs

```diff
@@ -113,16 +109,21 @@ impl<'s> Search<'s> {
         let mut new_query = self.query.clone();
         new_query.push_str(input.as_slice());
 
-        let mut results = SortedResultSet::new(self.visible_limit);
+        let mut heap = SortedResultSet::new(self.visible_limit);
```

Comment:
I think this rename loses some semantic value. `heap` refers to the implementation of a data structure not what it represents in this context